### DISCUSSION TITLE
Lesson 9 .env never set up

### DISCUSSION THREAD

**Discussion by DukeLurkin** [Cyfrin/foundry-full-course-cu#831] [Q&A]
I don't know how I missed the part where we added the .env file in this lesson but I did.  Can anyone let me know how to figure out the right sepolia URL and private key to add to the .env file?  I copied it how Patrick has it in the video but they must be different.  Thanks.

**Comment by theirrationalone** [comment]
Please, Follow Images Below 👇

![alchemy sepolia setup 1](https://github.com/Cyfrin/foundry-full-course-f23/assets/96755510/05c908f6-af86-461c-b4bf-b321520cde76)
![alchemy sepolia setup 2](https://github.com/Cyfrin/foundry-full-course-f23/assets/96755510/f61bde4e-6641-484e-9d1e-7ea5cdffa2ca)
![alchemy sepolia setup 3](https://github.com/Cyfrin/foundry-full-course-f23/assets/96755510/bc57b843-13b7-4023-a449-825da90f0e3c)
![alchemy sepolia setup 4](https://github.com/Cyfrin/foundry-full-course-f23/assets/96755510/aca78dc7-5e5b-45af-bb08-4f577d1209b2)
![alchemy sepolia setup 5](https://github.com/Cyfrin/foundry-full-course-f23/assets/96755510/78d99be1-77d7-4dad-83a0-61f56190bc16)
![alchemy sepolia setup 6](https://github.com/Cyfrin/foundry-full-course-f23/assets/96755510/b07c1567-27aa-4ef8-90e8-f0f03fffb64b)
![alchemy sepolia setup 7](https://github.com/Cyfrin/foundry-full-course-f23/assets/96755510/5e531326-c590-4404-8937-1c458d91811e)
![alchemy sepolia setup 8](https://github.com/Cyfrin/foundry-full-course-f23/assets/96755510/0fced4c5-e4fe-4d00-a83f-bc96f7193229)
![alchemy sepolia setup 9](https://github.com/Cyfrin/foundry-full-course-f23/assets/96755510/37c5c8da-8895-43e3-baf1-9f7374f299e2)
![alchemy sepolia setup 10](https://github.com/Cyfrin/foundry-full-course-f23/assets/96755510/8d6ccdc3-f5fe-428b-93ee-f964427c2b49)
